tests/test_import.py

```diff
@@ -0,0 +1,33 @@
+import unittest
+import builtins
+from importlib import reload
+
+import_orig = builtins.__import__
+
+
+def mocked_import(name, *args):
+    if name == "graypy":
+        raise ImportError()
+    return import_orig(name, *args)
+
+
+builtins.__import__ = mocked_import
```

Comment:
would it make more sense to have this in a `setUp` ? Now lets say we create another test case, the `tearDown` will undo this setup, and the setup will only be applied to the first testcase. A bit unexpected behavior.